doc/usage/getting_started.md

```diff
@@ -16,7 +16,7 @@ The procedure to provide credentials to access the cloud provider object store v
 
 * For `Azure Blob storage`, `STORAGE_ACCOUNT` and `STORAGE_KEY` should be made available as environment variables.
 
-* For `Openstack Swift`, `OS_USERNAME`, `OS_PASSWORD`, `OS_AUTH_URL`, `OS_TENANT_ID` and `OS_DOMAIN_ID` should be made available as environment variables.
+* For `Openstack Swift`, `OS_USERNAME`, `OS_PASSWORD`, `OS_AUTH_URL`, `OS_TENANT_ID` and `OS_DOMAIN_ID` or `OS_CLOUD` should be made available as environment variables.
```

Comment:
Who picks up `OS_CLOUD` env var? I don't see gophercloud API supporting this, atleast in our vendored files. Am I missing something?

Reply:
It does support it. In-fact we were using it before etcd-druid integration in gardener. In webhook for openstack.